Nodejs/Product/TestAdapter/TestFrameworks/Jest/jest.js

```diff
@@ -116,5 +123,36 @@ function logError() {
     console.error.apply(console, errorArgs);
 }
 
+function readConfigs(projectFolder, context)
+{
+    // First look for Jest.config.js, otherwise look at the package.json under "jest" tag
+    var userConfig;
+    const jestConfigPath = projectFolder + "\\jest.config.js";
+    const packageJsonPath = projectFolder + "\\package.json";
+
+    if(fs.existsSync(jestConfigPath))
```

Comment:
I'm guessing the `jest.conf.js` file takes precedence over `package.json` by default. Is it the same behavior with plane Jest?

I mean, does it merge both configs if `package.json` and `jest.conf.js` have settings?